### PR TITLE
Country pages, related content improvements

### PR DESCRIPTION
**Pull request by eadpearce**
https://uktrade.atlassian.net/browse/CMS-650

screencapture-exred-trade-great-8007-prototype-markets-asia-pacific-2018-12-17-14_33_41
screencapture-exred-trade-great-8007-prototype-markets-asia-pacific-australia-2018-12-17-14_38_47
screencapture-exred-trade-great-8007-prototype-markets-asia-pacific-australia-exporting-to-australia-2018-12-17-14_33_21

### CODE REVIEW

File: core/templates/core/campaign.html [hunk 1]
```diff
@@ -121,41 +120,15 @@ <h2 id="related-content-heading" class="heading-large margin-top-60">{{ page.rel
...
-          {% endif %}
-          <a id="related-page-two-link" href="{{ page.related_page_two_url }}" class="heading-medium grey-text margin-top-30">{{ page.related_page_two_heading }}</a>
-          {% if page.related_page_two_description %}
-            <p id="related-page-two-description">{{ page.related_page_two_description }}</p>
+      {% for article in page.related_pages %}
+        <li class="column-third" id="related-page-{{ article.meta.slug }}">
+          {% if article.article_image_thumbnail %}
+            <img id="related-article-{{ article.meta.slug }}-image" src="{{ article.article_image_thumbnail.url }}" alt="">
           {% endif %}
+          <a id="related-page-{{ article.meta.slug }}-heading" href="/prototype{{ article.full_path }}" class="heading-medium grey-text margin-top-30">{{ article.article_title }}</a>
```
Comment (richtier) [inline core/templates/core/campaign.html]: looks like non-prototype pages are linking to prototype pages: this template is used by `CampaignPageView`.

is that ok?

File: prototype/mixins.py [hunk 2]
```diff
@@ -4,24 +4,28 @@
...
-        response = cms_api_client.lookup_by_full_path(
-            full_path=unprefix_prototype_url(self.request.path),
-            draft_token=self.request.GET.get('draft_token'),
-        )
-        return handle_cms_response(response)
+class Breadcrumbs:
 
     def get_context_data(self, *args, **kwargs):
+        parts = unprefix_prototype_url(self.request.path).split('/')
+        url_fragments = [part for part in parts if len(part) > 0]
```
Comment (richtier) [inline prototype/mixins.py]: `[part for part in parts if part]` :boom:

File: core/templates/core/campaign.html (hunk 1 above)
Comment (eadpearce) [reply core/templates/core/campaign.html]: yeah for now. it's a separate PR to remove all the prototype prefixing

File: prototype/mixins.py (hunk 2 above)
Comment (eadpearce) [reply prototype/mixins.py]: yeah i tried that and for some reason the first item that came out was always an empty string 🤷‍♀️

Comment (eadpearce) [reply prototype/mixins.py]: obviously, as soon as I check it again it's working perfectly LOL. I will update this

**Pull request merged**